### ISSUE TITLE
Unable to execute standalone trail servers

### ISSUE THREAD

**Issue by tegamckinney** [nearform/trail#42]
When following the guides to run the standalone trail servers (fastify or hapi) using the following commands:
- `npm install @nearform/trail`
- `npx run trail-fastify-server`

The following errors is returned:
```
/Users/tmckinney/.nvm/versions/node/v12.18.2/bin/trail-fastify-server: line 1: use strict: command not found
/Users/tmckinney/.nvm/versions/node/v12.18.2/bin/trail-fastify-server: line 3: //: is a directory
/Users/tmckinney/.nvm/versions/node/v12.18.2/bin/trail-fastify-server: line 4: const: command not found
/Users/tmckinney/.nvm/versions/node/v12.18.2/bin/trail-fastify-server: line 6: syntax error near unexpected token `'unhandledRejection','
/Users/tmckinney/.nvm/versions/node/v12.18.2/bin/trail-fastify-server: line 6: `process.on('unhandledRejection', (err) => {'
```

I noticed when installing the package, the shebang is striped from the start.js file and thus is not properly executing.  I also noticed that if I clone the project locally and use npm link the shebang is preserved as expected.

Lastly, I noticed that the shebang for trail-core is also preserved whether install directly or as a dependency of the server projects.

Environment:
- Node v12.18.2
- NPM v6.14.6

**Comment by mcollina** [comment]
Have you tried the instructions in https://github.com/nearform/trail/tree/master/packages/trail-fastify-server#nearformtrail-fastify-server?

**Comment by tegamckinney** [comment]
@mcollina My apologies on the typo above however yes I did try the instructions at the link in your comment.  It's possibly a local system issue but I wasn't have troubles with any other commands as the DB init and setup npx commands worked without issues.  

When I used the commands at the link and took a look at `node_modules/.bin/trail-fastify-server` I noticed that the `#!/usr/bin/env node` missing.  When I clone and use npm link the `#!/usr/bin/env node` exists and the script runs as expected.
